config/configunmarshaler/defaultunmarshaler.go

```diff
@@ -187,6 +189,18 @@ func unmarshalService(srvRaw map[string]interface{}) (config.Service, error) {
 	return srv, nil
 }
 
+func defaultServiceTelemetryMetricsSettings() config.ServiceTelemetryMetrics {
+	addr := configtelemetry.GetMetricsAddr()
+	if addr == "" {
+		addr = configtelemetry.GetMetricsAddrDefault()
+	}
```

Comment:
Not clear this behavior, if user overwrite with empty why do we overwrite with default?

Reply:
A value is required here.  An empty address is invalid and thus I chose to treat it as the default value.  The other option would seem to be to panic here and I did not feel that appropriate.